### ISSUE TITLE
how to jump to target line

### ISSUE THREAD

**Issue by lemon0910** [brookhong/cscope.vim#29]
using the cscope.vim, the result will be listed in the window below, each line contain a result, how can I jump to the target result line, there seems no shortcut key

**Comment by brookhong** [comment]
go to the location list window, and locate a line, then just press `enter` will jump to the target.

**Comment by lemon0910** [comment]
yeah, I press enter key but it won't jump, Is there some common config that will make it disable? Thanks very much

**Comment by Muffo** [comment]
If I press `enter`, it will open the source file in the Location panel, instead in a new buffer beside the existing ones.

If press `o` (like I normally do in the quickfix window) the following error appears: **Cannot make changes, 'modifiable' is off**

Any idea?
